src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -208,6 +238,14 @@ export default {
 			this.needsNews = true;
 		}
 	},
+	mounted() {
+		if (this.passwordless && this.partnerContentId) {
+			this.fetchStrategicPartnerLoginInfoByPageId();
```

Comment:
Do we want to prefetch this data? Or are we not worried about image pop-in?

Reply:
The whole component won't show up until the data is available.

Reply:
That means the logos will pop-in and move the page elements around? Or is there no content below this in this use case?

Reply:
you could try to do this at the bottom of created...may need to wrap in nextTick if the passwordless and partnerContentId aren't getting set quickly enough...

Reply:
Actually yeah you're right the logos are popping in.
Prefetch implemented.  Done.